sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/processor/NoopLabelsProcessor.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.processor;
+
+import io.opentelemetry.api.metrics.common.Labels;
+import io.opentelemetry.context.Context;
+
+public class NoopLabelsProcessor implements LabelsProcessor {
```

Comment:
what about we make `create` return "nullable" value, and return null by default? That way we know if no processor is installed and we can optimize more in the accumulator (not even retrieving the current context, etc).

Reply:
I am not so sure about this one. Having nulls in public methods is usually considered an anti pattern, I do understand though that we have this call on a critical path for every metric recording. Seems like getting current context is pretty lightweight - just pulling it from thread local most of the time. Unless you insist I would avoid null's. If you do insist I am happy to change though

Reply:
you are correct, probably we need to expose just the noop instance, so at least we can do the trick if we want to compare with noop reference which should be same as comparing with null.